notebooks/line-changes.ipynb

```diff
@@ -403,7 +403,7 @@
        "      <th>0</th>\n",
        "      <td>2</td>\n",
        "      <td>2</td>\n",
-       "      <td>Downtown LA - Westwood via Sunset Bl</td>\n",
+       "      <td>USC - Westwood via Sunset Bl</td>\n",
```

Comment:
GO BRUINS!

Reply:
Checked for linked pdf on 70 line changes and the all changes page:
![image](https://user-images.githubusercontent.com/8574425/146848272-95e471e5-66f4-4a5c-9707-78128987d15f.png)